EosGround/sendCommand.py

```diff
@@ -21,20 +25,49 @@ def send_command():
 
     cursor.execute(
         """
-        INSERT INTO transmit_table (packet_type, packet_sender, packet_priority, packet_destination, packet_generate_time, packet_body) VALUES 
-        (%s,%s,%s,%s,%s,%s)
+        INSERT INTO eos_schema.transmit_table (packet_type, sender, priority, destination, generate_time, body)
+        VALUES (%s,%s,%s,%s,%s,%s)
         """, (packet_type, packet_sender, packet_priority, packet_destination, packet_generate_time, packet_body)
     )
     conn.commit()
+    print("Ping command sent")
+
+
+def send_cutdown_command():
+    ans = input("Are you sure you want to trigger the cutdown?  (y/n):\t")
+    if ans.lower() == 'y':
+        packet_generate_time = datetime.datetime.now()
+        packet_sender = Device.GROUND_STATION_1
+        packet_type = Type.DATA
+        packet_priority = Priority.DATA
+        packet_destination = Device.MISC_RADIO_1
```

Comment:
Destination should be Device.MISC_ENGINEERING_2

Reply:
good catch